v2/support/requirements_only_sample_app/LICENSE

```diff
@@ -0,0 +1,48 @@
+Copyright 2012 Zendesk
```

Comment:
Probably because that's when the original license was created and we've been copying the same file ever since. Do we need to update them every year? This file is copied in some many places...